akka-http2-support/src/test/scala/akka/http/impl/engine/http2/Http2ServerSpec.scala

```diff
@@ -305,9 +305,7 @@ class Http2ServerSpec extends AkkaSpecWithMaterializer("""
 
       // Reproducing https://github.com/akka/akka-http/issues/2957
       "close the stream when we receive a RST after we have half-closed ourselves as well" in new WaitingForRequestData {
-        import akka.http.scaladsl.client.RequestBuilding._
         // Client sends the request, but doesn't close the stream yet. This is a bit weird, but it's whet grpcurl does ;)
-        val post = Post("/grpc.reflection.v1alpha.ServerReflection/ServerReflectionInfo")
```

Comment:
Some unrelated cleanup. Can you remember, @raboof if that's just a left-over?

Reply:
Looks like a leftover from a previous version of that test, yes :+1: